wear/src/main/java/info/nightscout/androidaps/watchfaces/RICTxWF01.java

```diff
@@ -0,0 +1,206 @@
+            basalBackgroundColor = ContextCompat.getColor(getApplicationContext(), R.color.basal_dark);
+            basalCenterColor = ContextCompat.getColor(getApplicationContext(), R.color.basal_light);
+            pointSize = 1;
+            setupCharts();
+        }
+
+        /* frame styles*/
+        LinearLayout mShapesElements = layoutView.findViewById(R.id.shapes_elements);
+        if (mShapesElements != null) {
+            String displayFormatType = (mShapesElements.getContentDescription().toString().startsWith("round") ? "round" : "rect");
+            String displayStyle=sharedPrefs.getString("rictxwf01_frameStyle", "full");
+            String displayFrameColor=sharedPrefs.getString("rictxwf01_frameColor", "red");
+            String displayFrameColorSaturation=sharedPrefs.getString("rictxwf01_frameColorSaturation", "500");
+            String displayFrameColorOpacity=sharedPrefs.getString("rictxwf01_frameColorOpacity", "1");
+
+            // Load image with shapes
+            String styleDrawableName = "rictxwf01_bg_" + displayStyle + "_" + displayFormatType;
+            try {
+                mShapesElements.setBackground(getResources().getDrawable(getResources().getIdentifier(styleDrawableName, "drawable", getApplicationContext().getPackageName())));
+            } catch (Exception e) {
+                Log.e("rictxwf01_frameStyle", "RESOURCE NOT FOUND >> " + styleDrawableName);
+            }
+
+            // set background-tint-color
+            if (displayFrameColor.equalsIgnoreCase("multicolor") || displayStyle.equalsIgnoreCase("none")) {
+                mShapesElements.setBackgroundTintList(null);
+            } else {
+                String strColorName =((   displayFrameColor.equals("white") || displayFrameColor.equals("black")  )?displayFrameColor:displayFrameColor+"_"+displayFrameColorSaturation);
+                Log.v("rictxwf01_strColorName",strColorName);
+                try {
+                    ColorStateList colorStateList = ContextCompat.getColorStateList(getApplicationContext(), getResources().getIdentifier(strColorName, "color", getApplicationContext().getPackageName()));
+                    mShapesElements.setBackgroundTintList(colorStateList);
+                } catch (Exception e) {
+                    mShapesElements.setBackgroundTintList(null);
+                    Log.e("rictxwf01_ColorName", "COLOR NOT FOUND >> " + strColorName);
+                }
+            }
+
+            // set opacity of shapes
+            mShapesElements.setAlpha(Float.parseFloat(displayFrameColorOpacity));
+
+        }
+
+        /* optimize font-size  --> when date is off then increase font-size of time */
+        Boolean isShowDate = sharedPrefs.getBoolean("show_date", false);
+        if (!isShowDate) {
+            layoutView.findViewById(R.id.date_time).setVisibility(View.GONE);
+            mHour.setTextSize(62);
```

Comment:
This magic number is only provide a text size, if date function is user defined OFF. Because it's very dependence from watch face, I will let be unchanged.